style/components/navbar.css

```diff
@@ -0,0 +1,29 @@
+.menu {
+  display: flex;
+  height: 100%;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  overflow: hidden;
+  box-shadow: rgba(0, 0, 12, 0.24) 0 -3px 8px;
+}
+
+.menu__item {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
```

Comment:
`cursor: pointer` is originally meant to be used with links and not with buttons. You decide (like GitHub did when they started using `cursor: pointer` everywhere).